ledger/src/blockstore_db.rs

```diff
@@ -508,7 +508,7 @@ impl Rocks {
     ///
     /// Full list of properties that return int values could be found
     /// [here](https://github.com/facebook/rocksdb/blob/08809f5e6cd9cc4bc3958dd4d59457ae78c76660/include/rocksdb/db.h#L654-L689).
-    fn get_int_property_cf(&self, cf: &ColumnFamily, name: &str) -> Result<i64> {
+    fn get_int_property_cf(&self, cf: &ColumnFamily, name: &'static std::ffi::CStr) -> Result<i64> {
```

Comment:
Was this explicitly required by the version upgrade?

Reply:
Yep, looks like rust-rocksdb has changed its type that causes the compile error:

```
error[E0308]: mismatched types
    --> ledger/src/blockstore_db.rs:1002:35
     |
1002 |                 .get_int_property(RocksProperties::BLOCK_CACHE_USAGE)
     |                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `str`, found struct `std::ffi::CStr`
     |
     = note: expected reference `&str`
                found reference `&'static std::ffi::CStr`
```

But since we always use the predefined rocksdb properties, the change should be fine to us.